Makefile

```diff
@@ -327,14 +327,26 @@ build-qe-napi:
 	cargo build --package query-engine-node-api --profile $(PROFILE)
 
 build-qe-wasm:
-ifndef $(NIX)
+ifneq ($(strip $(NIX)),)
```

Comment:
Rewier's note: "The strip function is used to remove leading and trailing whitespace from the value of the variable". I'm not quite sure why this is needed, but it's safe to keep.

Reply:
this is needed because depending on your shell `command -v nix` (Which is used to populate the `NIX` variable) might not return what you want for the comparision to be effective.